Packages/Fluxity/Editor/HistoryWindow.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace AIR.Fluxity.Editor
+{
+    class HistoryWindow : FluxityRuntimeEditorWindow
+    {
+        private const int RECENT_CAPACITY = 30;
+        private const string TIME_FORMAT = "HH:mm:ss.fff";
+        private readonly Queue<DispatchData> _recentDispatchHistory = new Queue<DispatchData>(RECENT_CAPACITY);
+        private Vector2 _panelScrollViewPos;
+        private bool _subscribed = false;
+        private IDispatcher _dispatcher;
```

Comment:
Use just the GetDispatcher if possible.

Looks like this is only used inside 1 method so is not needed.